graphene_django/converter.py

```diff
@@ -17,7 +18,12 @@
 
 
 def convert_choice_name(name):
-    return to_const(force_text(name))
+    name = to_const(force_text(name))
+    try:
+        assert_valid_name(name)
+    except AssertionError:
+        name = "A_%s" % name
```

Comment:
This could break badly if python is run with optimisations on (`python -O`) because that disables `assert` statements, as used in `assert_valid_name`. The possible fixes are to update `assert_valid_name` to raise a different exception (`ValueError` probably) or to add a parallel function to `assert_valid_name` for use like this.